README.md

```diff
@@ -55,383 +51,208 @@ of the [Federated Identity Community Group](https://fedidcg.github.io/).
 
-The browser looks into the credential store and sees that there is a credential this is effective for `example.com` from `login.idp.net`.
-The browser give the user a "credential chooser" UI that allows them to share their account at `login.idp.net` with `example.com`.
-Once the user consents, a link is made and the Promise is resolved with a Credential.
-
-### Scenario 4: User intends to link to an identity provider they are already logged in to, but the relying party cannot provide the origin of
-
-In this scenario, the user is already logged into an identity provider that the relying party is willing to accept, but may not be willing or able to provide the origin of.
-This may be because the relying party trusts a class of identity providers with voluntary membership (e.g., IndieAuth), or because they do not wish to provide a list of acceptable identity providers to the browser (e.g., a consortium with anonymous membership).
-To request a credential in this way, the relying party needs to specify a provider with a given "type", like so:
+This could also be accomplished client-side only by passing a list of approved_clients into the login status API call:
 
 ```js
-let credential = await navigator.credentials.get({
-  'identity' : {
-    'providers' : [
-      {
-        type : "example-string-to-match",
-      },
-    ]
-  }
+navigator.login.setStatus("logged-in", {
+	accounts: [{
+         // ... other fields
+         approved_clients: ["https://rp.example"],
+         // ... other fields
+      }]
 });
 ```
 
-## Relying Party API, Getting a Credential
-
-The site that the user wants to log into needs to call the already existing method `navigator.credentials.get()`. We put our arguments under the `identity` key in the options argument, as does FedCM.
-While not shown here, this can be combined with arbitrary other credential arguments.
+The prompt also takes the RPs Privacy Policy and Terms of Service, which would normally be passed via the client_metadata_endpoint, but in the absence of one, could be passed in the JS call by the RP. This has also been [suggested in discussions around the IdP Registration API](https://github.com/w3c-fedid/idp-registration/issues/8).
 
 ```js
-let credential = await navigator.credentials.get({
-  identity : {
-    providers : [
-      {
-        origin: "https://login.idp.net",
-        loginURL: "https://bounce.example.com/?u=https://login.idp.net/login.html?r=https://rp.net/",
-        loginTarget: "redirect",
-      },
-    ]
-  }
+const {profile: {id}} = await navigator.credentials.get({
+  identity: {
+        providers: [{url: "https://example.com"}],
+        rp: {
+          privacyPolicy: "https://rp.example/pp.html",
+          termsOfService: "https://rp.example/tos.html",
+        }
```

Comment:
@ekovac This is what I meant

Reply:
ack. I've added some details about this to the PR.